Makefile

```diff
@@ -165,8 +166,13 @@ build-benchmark-docker: ## Build benchmark test docker image for NGINX Plus, nee
 		-f test/docker/Dockerfile .
 
 # Install tests
-test-install: ## Run agent install test
-	GOWORK=off CGO_ENABLED=0 go test -v ./test/install
+test-install: ## Run agent install/uninstall test for release
+	AGENT_PACKAGE_FILE=${shell find ./build/packages/deb/*${OS_CODENAME}_${LOCAL_ARCH}.deb -type f -exec realpath {} \;} \
```

Comment:
Think we should let the pipeline dictate where the AGENT_PACKAGE_FILE variable is?